cherryontech/JourneyWise
# implimented feature button for https://marisolsaumi.atlassian.net/bro…

## Pull Request

### Description
Implemented features/changes for [JW-59](https://marisolsaumi.atlassian.net/browse/JW-59) and [JW-62](https://marisolsaumi.atlassian.net/browse/JW-62). This pull request introduces the following changes:
- Implemented a new Button component.
- Added Tailwind CSS for styling.
- Updated the enlist file.
- Added the landing page.
- Updated Prettier configuration for code formatting consistency.

### Related Issue
[Cite any related issues this PR addresses. For example, "Fixes #123" or "Resolves #456".]

### Description of Changes
* The Button component provides a reusable button element throughout the application, enhancing user interaction.
* Tailwind CSS is utilized to streamline styling and maintain consistency across components.
* The enlist file is updated to include the new Button component for integration into the application.
* Prettier configuration is updated to ensure consistent code formatting practices.

### Checklist
- [x] Read the CONTRIBUTING.md document.
- [x] Follow code style of this project.
- [ ] Add suitable test coverage for the changes made.
- [x] Ensure changes generate no new warnings or errors.
- [x] Update documentation or comments where necessary.
- [ x] Ensure all tests pass locally.

### Testing Instructions
* To test the new Button component, navigate to [specific page/location] and observe its behavior

### Screenshots (if applicable)
[Add screenshots to help explain your changes, if applicable.]

### Gotchas/Learnings
* While implementing the Button component, I learned about the importance of consistent styling and how Tailwind CSS can help maintain a unified design system across the application.

### Additional Notes (Optional)
* Include any other relevant information for reviewers.
